### DISCUSSION TITLE
Ship software-properties-kde on KDE images

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#71] [General]
Just a minor nitpick, but software-properties-kde could be shipped instead of the gtk frontend on KDE and lxqt images.

Apart from that, I really, really like this distro as it ships with everything I need, and it is now what I recommend to people instead of Linux mint. Thank you so much for making this!

**Comment by geckolinux** [comment]
Hi there, thanks a lot for the kind feedback!
Regarding `software-properties-kde` the thing is that Synaptic will still be present in the Plasma edition, and the software-properties can also be accessed from within Synaptic. As far as I know `software-properties-kde` won't open from within Synaptic even if it's installed, right?

**Reply by ghost** [reply]
Hi, sorry for the late response. No, I don't think Synaptic can use software-properties-gtk, but Muon (which is essentially the same as Synaptic but in qt) _does_ use software-properties-kde. I don't know how much of the system depends on Synaptic, but it may be possible for you to replace it with Muon?
Thank you for your time!

**Reply by geckolinux** [reply]
Hi @Azl-Eyekay , thanks for the feedback. I haven't heard very good things about Muon, and it's not as mature as Synaptic. The SpiralLinux Plasma edition doesn't try to be a pure KDE-only system, it simply chooses the best tool for the job. In the case of Synaptic it is used across the board because it's reliable and because the language support script and the SpiralLinux update/upgrade instructions apply to all the editions. However, feel free to add/remove programs on your personal installation to suit your needs and preferences.
Thanks again!

**Reply by ghost** [reply]
Ok, I see. Thanks for your time and such a good distro!
